src/main/java/ArticlePage.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class ArticlePage {
+
+    WebDriver driver;
+
+    public ArticlePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void chooseFirstArticle(){
```

Comment:
can you create a generic function like this:
`chooseProduct(int index)` instead of choosing always the first product?